pyvisa-py/gpib.py

```diff
@@ -41,10 +39,9 @@ def _find_listeners():
 
 StatusCode = constants.StatusCode
 
-# linux-gpib timeout constants, in milliseconds. See self.timeout.
-TIMETABLE = (0, 1e-2, 3e-2, 1e-1, 3e-1, 1e0, 3e0, 1e1, 3e1, 1e2, 3e2, 1e3, 3e3,
-             1e4, 3e4, 1e5, 3e5, 1e6)
-
+# linux-gpib timeout constants, in milliseconds. See GPIBSession._set_timeout.
```

Comment:
Those numbers are in second now no ?

Reply:
yes, i forgot to change comment properly